opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -124,48 +125,133 @@ message MetricDescriptor {
   // described by http://unitsofmeasure.org/ucum.html.
   string unit = 3;
 
-  // Type is the type of values a metric has.
-  enum Type {
-    // INVALID_TYPE is the default Type, it MUST not be used.
-    INVALID_TYPE = 0;
+  // MeasurementValueType determines the value type for a measurement.
```

Comment:
This may determine whether Int64DataPoint or DoubleDataPoint is used. 

TODO: There is an open question about whether this should control int64 vs. double for Histogram and Summary. There are good arguments on both sides of this.

Reply:
Added the TODO. We should discuss, also once this PR is merged will make sure we have an issue for every TODO in the code to keep track. Not creating issues now since this is not approved.